R/register-chunk-hook.R

```diff
@@ -0,0 +1,53 @@
+#' Use vcr in vignettes
+#'
+#' @description
+#' `setup_knitr()` registers a `use_cassette` hook so that you map
+#' a vcr cassette to a knitr chunk. Call it in your setup chunk:
+#'
+#' ````
+#' ```{r setup}
+#' #| include: false
+#' vcr::setup_knitr()
+#' ```
+#' ````
+#'
+#' Then in a chunk where you want to use a cassette, set the `cassette` chunk option
+#' to the name of the cassette:
+#'
+#' ````
+#' ```{r}
+#' #| cassette: name
+#' req <- httr2::request("http://r-project.org")
+#' resp <- httr2::req_perform(req)
+#' ```
+#' ````
+#'
+#' @param prefix An prefix for the cassette name to make cassettes unique across
+#'   vignettes. Defaults to the file name (sans extension) of the currently
```

Comment:
```suggestion
#'   vignettes. Defaults to the file name (without extension) of the currently
```

As much as I like French words I think they're less accessible. https://books.google.com/ngrams/graph?content=without%2C+sans&year_start=1800&year_end=2022&corpus=en&smoothing=3